src/microbe_stage/editor/CellEditorComponent.cs

```diff
@@ -1281,6 +1281,12 @@ public void OnRigidityChanged(int desiredRigidity)
             return;
         }
 
+        if (Editor.MutationPoints == 0 && cost == 0)
```

Comment:
I think that it would be good to add a comment here clarifying why this check is needed (if I get it right, it handles the corner case of the cost being rounded to zero, while adding up with other costs in such a way that MP ends up at -1)